### PR TITLE
Add missing required db folder and fix docker-compose

### PR DESCRIPTION
**Pull request by se-bastiaan**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ services:
             - 8000:8000
         volumes:
             - ./:/usr/src/app/
+            - ./db:/usr/src/app/db
```
Comment (joren485) [inline docker-compose.yml]: Is this necessary if `./:/usr/src/app/` is already specified?

Comment (pingiun) [reply docker-compose.yml]: This makes sure the db folder is actually created, which settings.py now has for the default db folder

Comment (joren485) [reply docker-compose.yml]: In that case, the `.gitkeep` file is not necessary, right?

Comment (pingiun) [reply docker-compose.yml]: I just tested this locally and the .gitkeep and the db folder are not necessary with this change in the docker compose. Either change can thus be removed

**Comment by thomwiggers** [comment]
Why do you want to add `0.0.0.0` to `DJANGO_HOSTS`?

**Comment by joren485** [comment]
@thomwiggers `docker-compose.yml` runs the server on `runserver 0.0.0.0:8000`. Without adding `0.0.0.0` to the `ALLOWED_HOSTS` (which is taken from `DJANGO_HOSTS`), an error is shown.

**Comment by pingiun** [comment]
@joren485 you should view the website on localhost:8000, 0.0.0.0 just means the server will listen on all interfaces

**Comment by joren485** [comment]
@pingiun Ah of course, my bad.

**Comment by thomwiggers** [comment]
`ALLOWED_HOSTS` is indeed unrelated to the listen port (it only relates to HTTP headers).

File: docker-compose.yml (hunk 1 above)
Comment (se-bastiaan) [reply docker-compose.yml]: The folder is necessary if you don't use Docker. Without it and the .gitkeep `./manage.py migrate` outside a container won't work.

**Pull request merged**